bin/logstashScripts/logstash.conf

```diff
@@ -0,0 +1,37 @@
+input {
+
+file {
+path => "/home/alan/gitfolder/jersey/**/*.java"
```

Comment:
could you specify a more generic location, rather than your home
  - e.g. {ELK_SRC_HOME}

so that its clear to change the value